BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -63,16 +65,19 @@ class SideMenuFilterViewController: UITableViewController {
             slideValueLabel.text = "\(roundedNumber)"
         }
         
+        defaultPreferences.set(distanceFilter, forKey: "distanceFilter")
+        
     }
 
     @IBAction func openNowSwitchOffOn(_ sender: Any) {
 
-        if openNowSwitch.isOn {
+        if openNowSwitch.isOn == true {
```

Comment:
it was definitely better as `.isOn` instead of `isOn == true`. It is more concise code and natural to read as:
" If the switch is on" instead of "if the switch is on is a true statement".